{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError"},"sections":[],"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authError"}],"platforms":["macOS"]}],"kind":"declarations"}],"variants":[{"paths":["\/documentation\/yorkie\/doceventtype\/autherror"],"traits":[{"interfaceLanguage":"swift"}]}],"schemaVersion":{"minor":3,"major":0,"patch":0},"abstract":[{"type":"codeVoice","code":"authError"},{"type":"text","text":" indicates an authorization failure in syncLoop or watchLoop."}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/Yorkie\/documentation\/Yorkie","doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType"]]},"metadata":{"role":"symbol","roleHeading":"Case","modules":[{"name":"Yorkie"}],"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authError"}],"title":"DocEventType.authError","symbolKind":"case","externalID":"s:6Yorkie12DocEventTypeO9authErroryA2CmF"},"references":{"doc://Yorkie/documentation/Yorkie/DocEventType":{"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"DocEventType"}],"url":"\/documentation\/yorkie\/doceventtype","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType","title":"DocEventType","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"DocEventType","kind":"identifier"}],"abstract":[{"code":"DocEventType","type":"codeVoice"},{"type":"text","text":" is document event types"}],"kind":"symbol","type":"topic"},"doc://Yorkie/documentation/Yorkie":{"role":"collection","type":"topic","kind":"symbol","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie","abstract":[],"url":"\/documentation\/yorkie","title":"Yorkie"},"doc://Yorkie/documentation/Yorkie/DocEventType/authError":{"url":"\/documentation\/yorkie\/doceventtype\/autherror","identifier":"doc:\/\/Yorkie\/documentation\/Yorkie\/DocEventType\/authError","abstract":[{"type":"codeVoice","code":"authError"},{"type":"text","text":" indicates an authorization failure in syncLoop or watchLoop."}],"title":"DocEventType.authError","kind":"symbol","role":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"authError"}]}}}